aeron-client/src/main/c/aeronc.h

```diff
@@ -38,15 +38,20 @@ typedef struct aeron_header_stct aeron_header_t;
 #pragma pack(4)
 typedef struct aeron_header_values_stct
 {
-    int32_t frame_length;
-    int8_t version;
-    uint8_t flags;
-    int16_t type;
-    int32_t term_offset;
-    int32_t session_id;
-    int32_t stream_id;
-    int32_t term_id;
-    int64_t reserved_value;
+    struct aeron_header_values_frame_stct
```

Comment:
consider typedef this out?

Reply:
We typically don't add the typedefs if it is nested.  @tmontgomery Thoughts?

Reply:
I think this is fine.